onnxmltools/convert/lightgbm/_parse.py

```diff
@@ -90,3 +123,7 @@ def parse_lightgbm(model, initial_types=None, target_opset=None,
         raw_model_container.add_output(variable)
 
     return topology
+
+
+def default_initial_types(model):
```

Comment:
This function does not seem to be used anymore.

Reply:
Removed